tools/depends/download-files.include

```diff
@@ -18,6 +18,12 @@ ifeq ($(NATIVE_OS), osx)
   SED_FLAG = -i ''
 endif
 
+# non-depends builds might not set this, use defaults
+ifeq ($(HASH_TOOL),)
+  HASH_TOOL = sha512sum
+  HASH_TOOL_FLAGS = -c --status
+endif
+
 HASH_SUM = $($(shell echo $(HASH_TYPE) | tr  '[:lower:]' '[:upper:]'))
```

Comment:
I guess this makes part of the other PR unnecessary, but fallback to a sane set of defaults is probably best practice anyways.

Reply:
yes, in fact I've tested it with dav1d without the overides.
But we really just want sane defaults here, in case the override is missing somewhere